card/mod/core/set/all/actify.rb

```diff
@@ -53,10 +53,10 @@ def valid?(*)
   act { super }
 end
 
-def update_attributes *args
+def update *args
   act { super }
```

Comment:
Does `update_attributes` still work or is it now linked to the non-actified rails version?
Since rails supports `update_attibutes` I think we should support it, too.

Reply:
In rails, `update_attributes(!)` is an alias for `update(!)`, which they changed to be the more idiomatic method.  So since we now override `update`, it will still work to call `update_attributes` –– and you'll still get an act.  This is much preferable to the confusing current situation, in which `update_attributes` gets an act, but `update` doesn't.

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attributes

Reply:
That's not true. `update_attributes` is linked to rails' `update` method not the overridden one